src/styles/components/_tile.scss

```diff
@@ -149,11 +132,13 @@
 }
 
 .mc-tile-overlay {
+  transform: scale(1.01);
```

Comment:
saw the "peeking background" happening again so i added this back in

Reply:
As long as it doesn't go back to having the same issue we were trying to fix where it bleeds out of the parent container...

Reply:
This comment piqued my interest so I took another look and fixed it.